internal/config/config.go

```diff
@@ -21,6 +21,8 @@ type NeutronQueryRelayerConfig struct {
 	CheckSubmittedTxStatusDelay uint64                   `split_words:"true" default:"10"`
 	QueriesTaskQueueCapacity    int                      `split_words:"true" default:"10000"`
 	PrometheusPort              uint16                   `split_words:"true" default:"9999"`
+	TxRetryDuration             time.Duration            `required:"true" default:"120s"`
+	TxRetryRetries              uint                     `required:"true" default:"3"`
```

Comment:
Please make a corresponding PR in neutron repo, we have a `docker-compose.yaml` there which needs to be updated, and we also need a similar PR in neutron-integration.tests.

Reply:
https://github.com/neutron-org/neutron/pull/87

Reply:
And what about PR into integration tests?

Reply:
added